scripts/bitnami-kafka-docker-compose.yml

```diff
@@ -2,15 +2,15 @@ version: '2'
 
 services:
   zookeeper:
-    image: 'bitnami/zookeeper:3'
+    image: 'docker.io/bitnami/zookeeper:latest'
     ports:
       - '2181:2181'
     volumes:
       - 'zookeeper_data:/bitnami'
     environment:
       - ALLOW_ANONYMOUS_LOGIN=yes
   kafka:
-    image: 'bitnami/kafka:2'
+    image: 'docker.io/bitnami/kafka:latest'
     ports:
       - '9092:9092'
       - '29092:29092'
```

Comment:
Do we want to include this file into the source distribution/wheel? If so the corresponding data section should be filled in in the `setup.py`, and https://github.com/bluesky/bluesky-kafka/blob/master/MANIFEST.in should be updated.

Reply:
The docker-compose script is strictly for testing so I had not thought about including it in the source distribution. I am inclined to leave it out and assume if someone wants to run the tests they will clone the repository. Is that reasonable or would it be better to include this script in the source distribution?

Reply:
It's reasonable, let's leave it out.